docs/medical-api/api-reference/patient/create-patient.mdx

```diff
@@ -10,6 +10,15 @@ api: "POST /medical/v1/patient"
   The ID of the Facility where the Patient is receiving care.
 </ParamField>
 
+<ParamField query="externalId" type="string">
+  An external Patient ID to associate to a Patient in Metriport.
+</ParamField>
```

Comment:
`externalId` is part of the body, as per the schema definition on the SDK, this is defining a new query param

Reply:
Wdym? This is similar to facilityId which is also a param